build.assets/Dockerfile-centos7

```diff
@@ -92,6 +92,31 @@ RUN mkdir -p /opt && cd /opt && \
     cd /opt/libbpf-${LIBBPF_VERSION}/src && \
     scl enable devtoolset-11 "make && BUILD_STATIC_ONLY=y DESTDIR=/opt/libbpf make install"
 
+## LIBPCSCLITE #####################################################################
+
+FROM centos:7 AS libpcsclite
+
+# Install required dependencies.
+RUN yum groupinstall -y 'Development Tools' && \
+    yum update -y && \
+    yum -y install centos-release-scl-rh && \
+    yum install -y \
+        autoconf-archive \
+        libudev-devel \
+        scl-utils \
+        centos-release-scl \
+        devtoolset-11-gcc* && \
+    yum clean all
+
+# Install libpcsclite - compile with a newer GCC. The one installed by default is not able to compile it.
+ARG LIBPCSCLITE_VERSION
+RUN git clone --depth=1 https://github.com/gravitational/PCSC.git && \
```

Comment:
Can you create a tag in our fork and pin the version here? Otherwise, we won't know which version was used for each build.